controllers/grid/users/reviewer/form/traits/HasReviewDueDate.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @file controllers/grid/users/reviewer/form/traits/HasReviewDueDate.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class HasReviewDueDate
+ *
+ * @brief Helper trait to get the review submit and/or response due dates
+ */
+
+namespace PKP\controllers\grid\users\reviewer\form\traits;
+
+use Carbon\Carbon;
+use PKP\context\Context;
+
+trait HasReviewDueDate
+{
+    public const REVIEW_SUBMIT_DEFAULT_DUE_WEEKS = 4;
+    public const RESPONSE_RESPONSE_DEFAULT_DUE_WEEKS = 4;
```

Comment:
Probably should be named `REVIEW_RESPONSE_DEFAULT_DUE_WEEKS`. Also I see that it was 3 before, is it intentional?

Reply:
Right , fixed it .